src/shape BrokenLine.ts

```diff
@@ -0,0 +1,88 @@
+abstract class Shape {
```

Comment:
1. Rename
`shape BrokenLine.ts` -> `BrokenLine.ts`
`shape Circle.ts` -> `Circle.ts`
`shape Rectangle.ts` -> `Rectangle.ts`
2. Extract `abstract class Shape` into separate `Shape.ts` and import this class in `BrokenLine.ts`, `Circle.ts` and `Rectangle.ts`